internal/adapter/sqlite/note_dao_test.go

```diff
@@ -452,10 +452,25 @@ func TestNoteDAOFindMatch(t *testing.T) {
 	)
 }
 
+func TestNoteDAOFindMatchWithMultiMatch(t *testing.T) {
+	testNoteDAOFindPaths(t,
+		core.NoteFindOpts{
+			Match:         []string{"daily | index", "second"},
+			MatchStrategy: core.MatchStrategyFts,
```

Comment:
Did not test all strats here (do in tesh tests), but these seemed brief anyway (e.g. does not currently test stemming, etc). Didn't want to boat the file but can add at request.